python_bindings/src/halide/halide_/PyBuffer.cpp

```diff
@@ -324,7 +324,10 @@ void define_buffer(py::module &m) {
                 const int d = b.dimensions();
                 const int bytes = b.type().bytes();
                 std::vector<Py_ssize_t> shape, strides;
-                for (int i = 0; i < d; i++) {
+                // Halide's default indexing convention is col-major (the most rapidly varying index comes first);
+                // Numpy's default is row-major (most rapidly varying comes last).
+                // We want to reverse the order so that most-varying comes first, for better vectorization.
```

Comment:
nit: remove "for better vectorization" here and below.

Yes, we want to vectorize across the most-rapidly-varying dimension, which is the first in Halide's convention, but we could vectorize across the most rapidly varying dimension no matter where it showed up in the arg list.

Reply:
Done.